lib/swagger/jsonLoader.ts

```diff
@@ -122,6 +129,44 @@ export class JsonLoader implements Loader<Json> {
     if (skipResolveRef !== undefined) {
       cache.skipResolveRef = skipResolveRef;
     }
+
+    if (shouldDeference) {
+      cache.shouldResolve = shouldDeference;
+      await this.loadFile(cache);
+      //get unresolved content from cache.originalContent
+      const fileContent = JSON.parse(cache.originalContent!);
+      //remove unnecessary properties
+      removeProperty(fileContent);
+      //console.log(JSON.stringify(fileContent));
+
+      //resolve all refs using lib: https://github.com/APIDevTools/json-schema-ref-parser
+      const resolveOption: $RefParser.Options = {
+        resolve: {
+          file: {
+            canRead: true,
```

Comment:
It tries to resole file reference. How about the internal $ref? Is it resolved by default?